vars/Ubuntu-20.04.yml

```diff
@@ -0,0 +1,16 @@
+php_version: "{{ atom_php_version|default('7.4') }}"
```

Comment:
It's not a blocker, but I would just hardcode the php_version.  I don't see any advantage to making php_version a variable, but hardcoding the version the package names.

Reply:
Hi @djjuhasz - Good point! I have amended the commit to assign php_version to "7.4"